website/docs/docs/deploy/job-commands.md

```diff
@@ -0,0 +1,81 @@
+
+### Checkbox commands
+
+For every job, you have the option to select the [Generate docs on run](/docs/collaborate/build-and-view-your-docs) or [Run source freshness](/docs/deploy/source-freshness) checkboxes. These checkboxes enable you to generate project docs or enable source freshness automatically.
+
+**Job outcome**  &mdash; During a job run, if either "checkbox command" (source freshness or generate documentation) fails, the job can still be successful if all subsequent steps are successful. 
+
+### Command list
+
+You can add or remove as many [dbt commands](/reference/dbt-commands) as necessary for every job, however, you need to have at least one dbt command. 
+Commands labeled as "CLI only" in the [dbt Command reference doc](/reference/dbt-commands) are meant for [CLI](/docs/get-started/about-the-cli)-use only. In dbt Cloud, you can only run commands that _aren't labeled_ "CLI only".
+
+
+:::tip Using selectors
+
+Use [selectors](/reference/node-selection/syntax) as a powerful way to select and execute portions of your project in a job run. If a selector doesn't match any models, the job run won't consider it a failure. For example, to run tests for one_specific_model, use the selector: `dbt test --select one_specific_model`
+
+:::
+   
+    
+**Job outcome**  &mdash;  During a job run, the commands are “chained” together. This means if one of the run steps in the chain fails, then the next ones aren't executed. For example, if the 5th run step fails:
+(`dbt run --select state:modified+ --full-refresh --fail-fast`)
+then the subsequent commands aren't executed, and the entire job fails. The failed job returns a non-zero [exit code](/reference/exit-codes) and "Error" job status:
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/skipped-jobs.jpg" title="A failed job run that had an error during a run step"/>
+
+## Job command failures
+
+Job command failures can mean different things for different commands. Here are some reasons why a job command may fail:
+
+- **Failure at`dbt run`** &mdash; [`dbt run`](/reference/commands/run) executes compiled SQL model files against the current target database. It will fail if there is an error in any of the built models. Tests on upstream resources prevent downstream resources from running and a failed test will skip them.
+
+- **Failure at `dbt test`** &mdash;  [`dbt test`](/reference/commands/test) runs tests defined on models, sources, snapshots, and seeds. A test can pass, fail, or warn depending on its [severity](reference/resource-configs/severity). Unless you set [warnings as errors](/reference/global-configs#warnings-as-errors), only an error stops the next step. 
+
+- **Failure at `dbt build`** &mdash; [`dbt build`](/reference/commands/build) runs models, tests, snapshots, and seeds. This command is a series of commands in itself, partial failures. For example, if `dbt seed` is successful, but `dbt run` isn't, then it's treated as a failure with an Exit code 1.
```

Comment:
not sure i wou'd classify this as a superset of commands -- it's one command that properly sequences resources in DAG order (i.e. you wouldn't see a `dbt seed` and a `dbt run` anywhere in the logs.

"This command runs all resources in the DAG-specified order, and any failure on any upstream resource will cause a failure with an exit code 1 and all downstream resources will be skipped." or something